cluster/ci/monitoring/Makefile

```diff
@@ -33,7 +34,9 @@ prow-monitoring-deploy: ci-secret-deploy
 	#oc create -n prow-monitoring configmap grafana-dashboard-deck --from-file=./deck.json --dry-run -o yaml | oc apply -f -
 	#oc create -n prow-monitoring configmap grafana-dashboard-build-cop --from-file=./build_cop.json --dry-run -o yaml | oc apply -f -
 	#oc create -n prow-monitoring configmap grafana-dashboard-prow --from-file=./prow.json --dry-run -o yaml | oc apply -f -
-	#oc create -n prow-monitoring configmap grafana-dashboard-prow-components --from-file=./prow-components.json --dry-run -o yaml | oc apply -f -
+
+	# Fill prometheus-k8s basicAuthPassword in datasources file.
+	sed -i -e "s|CHANGE_ME|$(prometheus-k8s_token)|g" ./datasources.yaml
```

Comment:
I think we have to update the configMap `grafana-datasources` above.
Currently it is controlled by `config-updater` since no token/secret has been used so far.
And it does use now.

If we can ensure that this scrpit is always after `config-updater`. Then we can do the update there.
Otherwise, we need to disable the update from `config-updater` and only update here.

Reply:
we need to remove it from config_updater or we have to add the deploy to the postsubmit

Reply:
like you said :)

Reply:
#3948